tests/foreman/cli/test_contentcredentials.py

```diff
@@ -898,7 +709,7 @@ def test_positive_list(module_org):
         {'key': VALID_GPG_KEY_FILE_PATH, 'organization-id': module_org.id}
     )
     gpg_keys_list = ContentCredential.list({'organization-id': module_org.id})
-    assert gpg_key['id'] in [gpg['id'] for gpg in gpg_keys_list]
+    assert gpg_key['id'] in [gpg.get('id') for gpg in gpg_keys_list]
```

Comment:
Codewisely looks good to me, any idea why the assertion failed in PRT?
```
         gpg_keys_list = ContentCredential.list(****'organization-id': module_org.id})
 >       assert gpg_key['id'] in [gpg.get('id') for gpg in gpg_keys_list]
 E       AssertionError: assert '60' in ['36', '12', '43', '27', '37', '57', ...]
```
Does it pass locally?

Reply:
This test passes on its own but when run as a file fails intermittently, could the list of gpg keys returned by cli be too long?